example/android/app/src/debug/java/com/grabforminputexample/DetoxTest.java

```diff
@@ -0,0 +1,29 @@
+package com.grabforminputexample;
```

Comment:
"Detox requires that your project has a single dummy native Android test with some special content, which will be picked up by testRunner that you just added in the previous step, so let's create it now."

Reply:
https://wix.github.io/Detox/docs/introduction/project-setup#42-adding-an-auxiliary-android-test